nameserver/status.go

```diff
@@ -17,11 +17,7 @@ type EntryStatus struct {
 	Tombstone   int64
 }
 
-func NewStatus(ns *Nameserver, dnsServer *DNSServer) *Status {
-	if dnsServer == nil {
-		return nil
-	}
-
+func NewNSStatus(ns *Nameserver) *Status {
```

Comment:
I find the naming a bit confusing here. IMHO, `NewNSStatus` implies a nameserver status which I expect to  include upstream dns servers, ttl, etc.